projects/js-packages/publicize-components/src/components/media-section/styles.module.scss

```diff
@@ -0,0 +1,61 @@
+.wrapper {
+	padding: 0;
+
+	& :global( .components-spinner ) {
+		position: absolute;
+		top: 50%;
+		left: 50%;
+		margin-top: -9px;
+		margin-left: -9px;
+	}
+
+	// Stack consecutive buttons.
+	& :global( .components-button + .components-button ) {
+		display: block;
+		margin-top: 1em;
+	}
+
+	// This keeps images at their intrinsic size (eg. a 50px
+	// image will never be wider than 50px).
+	& :global( .components-responsive-wrapper__content ) {
+		max-width: 100%;
+		width: auto;
+		height: auto;
+	}
```

Comment:
Why do we need to do this? If there is an issue with the image picker component, can we fix it upstream? We shouldn't have small images here anyway. We could maybe deal with that when we work on validation.

Reply:
Ok, I see this is for the issue that @gmjuhasz. It's actually just the `width: auto` which fixes it. I feel like this is something that should be fixed in the component. I see that the featured image is patching it in a similar way!

Reply:
Yeah, we can reduce this to just `width: auto`, or at least that was working for me. I feel like we should open a PR on Gutenberg to fix this in the more general case.